### ISSUE TITLE
How to suppress export logs in open-telemetry agent?

### ISSUE THREAD

**Issue by VelDeveloper** [aws-observability/aws-otel-java-instrumentation#369]
I am using "aws-opentelemetry-agent" for my spring-boot application and we are exporting logs & traces via gRPC.
whenever it exports it generates a log line like below. These logs line blow up the logging system since we export them more often.
Can I suppress this log message? I want error logs whenever the export fails.

```
[2023-04-04T10:16:20.401Z] "POST /opentelemetry.proto.collector.trace.v1.TraceService/Export HTTP/2" 200 - via_upstream - "-" 2452 7 2 1 "-" "OTel-OTLP-Exporter-Java/1.23.1" "d3f20df4-13b0-956f-9202-72cd00fc3566" "xray-collector.opentelemetry-operator-system:4317" "10.10.200.164:4317" outbound|4317||xray-collector.sample.local - default

[2023-04-04T10:16:24.897Z] "POST /opentelemetry.proto.collector.metrics.v1.MetricsService/Export HTTP/2" 200 - via_upstream - "-" 23319 7 60 17 "-" "OTel-OTLP-Exporter-Java/1.23.1" "b8ed6dc9-c3ac-9f90-abef-e161e8282b2f" "prometheus-amp-collector.opentelemetry-operator-system:4317" "10.10.235.74:4317" outbound|4317||prometheus-amp-sample.svc.cluster.local  - default
```

**Steps to reproduce**
FROM maven:3.8.3-openjdk-17-slim

RUN mkdir -p /agent
RUN apt-get update && apt-get install -y wget
RUN wget -O /agent/aws-opentelemetry-agent.jar https://github.com/aws-observability/aws-otel-java-instrumentation/releases/latest/download/aws-opentelemetry-agent.jar

ENV JAVA_TOOL_OPTIONS="-javaagent:/agent/aws-opentelemetry-agent.jar \
-Dotel.resource.attributes=service.name=sample \
-Dotel.propagators=tracecontext \
-Dotel.traces.exporter=otlp \
-Dotel.exporter.metrics.endpoint=http://sample:4317 \
-Dotel.exporter.traces.endpoint=http://sample:4317 \
-Dotel.metrics.exporter=otlp"

VOLUME /srv/logs

ARG JAR_FILE
COPY ${JAR_FILE} sample.jar
ENTRYPOINT ["java","-jar","/sample.jar"]

**What version and what artifacts are you using?**
we are using the latest aws-opentelemetry-sdk

**Environment**
Environment: AWS EKS

I have created an issue [here](https://github.com/open-telemetry/opentelemetry-java/issues/5347) but they requested me to post it here.

**Comment by rapphil** [comment]
The logs that you posted seems like access logs generated in server side. Is your java workload running with more applications in the same pod?

If that is not the case, what is the logging configuration that you are using imn your spring application? can you share more details about that?

**Comment by VelDeveloper** [comment]
@rapphil I am using the spring-boot application, logging using logback but I am running this as a Java agent. As you can see from my DockerFile. We are deploying this application in AWS EKS using kustomize we configure the below flags while deploying for Otel agent,

            - name: OTEL_TRACES_EXPORTER
              value: otlp
            - name: OTEL_METRICS_EXPORTER
              value: otlp
            - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
              value: "trace exporter gRPC url"
            - name: OTEL_EXPORTER_OTLP_METRICS_ENDPOINT
              value: "metrics exporter gRPC url"
            - name: OTEL_RESOURCE_ATTRIBUTES
              value: "service.name=sample,service.env=dev"
            - name: OTEL_PROPAGATORS
              value: "xray"
            - name: OTEL_METRIC_EXPORT_INTERVAL
              value: "30000"
              
We see the above logs whenever it exports data. Do we have any way to set it as ERROR so that I want to see logs when it couldn't able to export the metrics/traces?

**Comment by rapphil** [comment]
How are you connecting the application to the collector? Can you provide more details about the deployment? 

Is the collector deployed and exposed as a service or are you doing something else?


The logs that you provided look very much like envoy/istio log lines: https://istio.io/latest/docs/tasks/observability/logs/access-log/#default-access-log-format

**Comment by VelDeveloper** [comment]
@rapphil Yes, you are correct thanks a lot for the reply. As you mentioned it is coming from the ISTIO envoy proxy. We will fix it and thanks again for your precious time. Please close the ticket.
